app/store/networkDrawer.ts

```diff
@@ -0,0 +1,51 @@
+import { CTransactionSegWit } from '@defichain/jellyfish-transaction'
+import { createSelector, createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+export interface Transaction {
+  broadcasted: boolean
+  signed: CTransactionSegWit
+  title?: string
+}
+
+export interface NetworkDrawerState {
+  transactions: Transaction[]
+  height: number
+  err?: Error
+}
+
+const initialState: NetworkDrawerState = {
+  transactions: [],
+  height: 49,
+  err: undefined
+}
+
+export const networkDrawer = createSlice({
+  name: 'networkDrawer',
+  initialState,
+  reducers: {
+    setHeight: (state, action: PayloadAction<number>) => {
+      state.height = action.payload
+    },
+    queueTransaction: (state, action: PayloadAction<Transaction>) => {
+      state.transactions = [...state.transactions, action.payload]
+    },
+    closeNetworkDrawer: (state) => {
```

Comment:
Where is this applied? This will force cancel ongoing task, can avoid it if not necessary. I think even `popTransaction` should be consumed by NetworkDrawer only.

Reply:
It's applied on the "OK" button if it's the last transaction of the stack.